tasklist/webapp/src/main/java/io/camunda/tasklist/webapp/security/se/SearchEngineUserDetailsService.java

```diff
@@ -86,10 +86,10 @@ public void initializeUsers() {
     }
   }
 
-  private boolean userExists(final String userId) {
+  private boolean userExists(String userId) {
```

Comment:
Minor question, any reason for this not being final for userId?

Reply:
I think it's just an automatic revert of previous commits. That's why final modifier is also reverted.